content/docs/2.4/scalers/azure-storage-queue.md

```diff
@@ -68,6 +70,8 @@ spec:
       #
       # Optional
       queueLength: "5" # default 5
+      # Optional, default: "queue.core.windows.net"
+      endpointSuffix: queue-endpoint-suffix
```

Comment:
Can we use a real example of the non-public cloud?

Reply:
That makes more sense :) Used the AzureUSGovernment endpoint for Queue and the AzureGermanCloud endpoint for Blob.